tensorflow/contrib/seastar/seastar_stat.h

```diff
@@ -0,0 +1,28 @@
+#ifndef TENSORFLOW_CONTRIB_SEASTAR_SEASTAR_STAT_H_
+#define TENSORFLOW_CONTRIB_SEASTAR_SEASTAR_STAT_H_
+#include <cstddef>
+#include <sys/time.h>
```

Comment:
This file is Linux specific. If you are only using ``gettimeofday``, see my note above.